at_secondary/at_secondary_server/lib/src/verb/manager/response_handler_manager.dart

```diff
@@ -40,8 +40,8 @@ class DefaultResponseHandlerManager implements ResponseHandlerManager {
       return _streamHandler;
     } else if (verb is NotifyAll) {
       return _notifyAllHandler;
-    } else if (verb is Sync) {
```

Comment:
for sync..baseresponse handler will be used?

Reply:
Yes Murali, for sync BaseResponseHandler will used.